packages/inference/test/HfInference.spec.ts

```diff
@@ -1406,4 +1406,50 @@ describe.concurrent("HfInference", () => {
 		},
 		TIMEOUT
 	);
+	describe.concurrent(
+		"Cerebras",
+		() => {
+			const client = new HfInference(env.HF_CEREBRAS_KEY);
```

Comment:
```suggestion
			const client = new HfInference(env.HF_CEREBRAS_KEY ?? "dummy");
```

(follow-up after merging https://github.com/huggingface/huggingface.js/pull/1226)